### ISSUE TITLE
Heatmap - group multiple runs by source and component

### ISSUE THREAD

**Issue by psegedy** [ibutsu/ibutsu-server#109]
We often run parallel and sequential set of tests in a single jenkins job. The problem is that both runs get different `run id` and the heatmap in dashboard is showing result just for one of runs.

**Comment by psegedy** [comment]
@john-dupuy any plans on adding this? It's still not possible to see results e.g. from prod-test-suite if you have some parallel and some sequential tests because these test runs have 2 run ids.

**Comment by john-dupuy** [comment]
@psegedy yeah it's been a while and I need to come back to it. IIRC it requires some hacky things to be done in the heatmap on both the BE and the FE.
